src/bidiMapper/domains/network/NetworkStorage.spec.ts

```diff
@@ -34,12 +34,6 @@ describe('NetworkStorage', () => {
     networkStorage = new NetworkStorage(eventManager);
   });
 
-  it('disposeRequestMap', () => {
```

Comment:
replace the test?

Reply:
Best we can do is pass in a mock Request and see if its dispose gets called. Other then that it's a private methods so we can't poke.